examples/gno.land/p/nt/avl/node.gno

```diff
@@ -108,12 +108,15 @@ func (node *Node) Get(key string) (index int, value any, exists bool) {
 // GetByIndex retrieves the key-value pair of the node at the given index
 // in the subtree rooted at the node.
 func (node *Node) GetByIndex(index int) (key string, value any) {
+	if index < 0 {
+		panic("GetByIndex: negative index not allowed")
+	}
+
 	if node.height == 0 {
-		if index == 0 {
-			return node.key, node.value
-		} else {
+		if index != 0 {
 			panic("GetByIndex asked for invalid index")
```

Comment:
Same here
nitpick: I think it is better you can have this change reverted to avoid reviewing uneccessary changes